### PR TITLE
Only Googlebot can access doi.org

### PR DESCRIPTION
**Pull request by bklaing2**
## Purpose

closes: _Add github issue that originated this PR_

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -1,7 +1,14 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
-#
-# To ban all spiders from the entire site uncomment the next two lines:
-# User-agent: *
-# Disallow: /
+
+User-agent: tGooglebot
```
Comment (kaysiz) [inline public/robots.txt:3]: @bklaing2 could this be a typo, the `t` at the beginning of the UserAgent

**Review by kaysiz — CHANGES_REQUESTED** [review]

File: public/robots.txt (hunk 1 above)
Comment (bklaing2) [reply public/robots.txt:3]: Oops yes, thank you!

**Review by kaysiz — APPROVED** [review]

**Review by jrhoads — APPROVED** [review]

**Comment by richardhallett** [comment]
Apologies I missed this PR and did the same:
https://github.com/datacite/akita/pull/369
They are functionally equivalent although I do not explicitly deny adbot, but this is covered by the global ignore.

**Comment by bklaing2** [comment]
> Apologies I missed this PR and did the same: #369 They are functionally equivalent although I do not explicitly deny adbot, but this is covered by the global ignore.

Well now I just noticed this update from @richardhallett, so it seems this may not address the bot problem as much as we hoped?

**Comment by richardhallett** [comment]
I forgot I did this, okay well in which case, mostly google that affect us. I guess just leave it for now then.

**Comment by jrhoads** [comment]
@bklaing2  Looks like we can close this then.

**Pull request closed without merging**